api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -151,3 +169,84 @@ message Telemetry {
   //          returned.
   bytes frame_header = 6;
 }
+
+//----------------------------------------------------------------------------------------------
+// Scheduling APIs.
+//----------------------------------------------------------------------------------------------
+
+// Request for the `ListGroundstationPlans` method.
+message ListGroundStationPlansRequest {
+  // The ID of the ground station to list plans for. The ID can be found on the StellarStation
+  // Console page for the ground station.
+  string ground_station_id = 1;
+
+  // The start time of the range of plans to list (inclusive). Only plans with an Acquisition of
+  // Signal (AOS) at or after this time will be returned. It is an error for the duration between
+  // `aos_after` and `aos_before` to be longer than 31 days.
+  google.protobuf.Timestamp aos_after = 2;
+
+
+  // The end time of the range of plans to list (inclusive). Only plans with an Acquisition of
```

Comment:
Isn't it exclusive? I think it is easier to make either of start time or end time exclusive.

Reply:
Good point, made endpoint exclusive.